terraform/helm/aptos-node/files/test-data/keys.json

```diff
@@ -0,0 +1 @@
+{"operator":{"data":"GetResponse","last_update":1651079195,"value":"0x2dd3e57f460a7c89c052ef88110b320379d79f76d8b24abcbe99335991e71b16"},"owner_account":{"data":"GetResponse","last_update":1651079195,"value":"f85fd00e30cd388d2609db24d6cc44f2652f27413456938eaa059f65231647cc"},"fullnode_network":{"data":"GetResponse","last_update":1651079195,"value":"0xd77ad0a95ad0e3a057318235aa363242a291a81ec08dc0d1ab0793f55c08b422"},"owner":{"data":"GetResponse","last_update":1651079195,"value":"0x6953493de50e9b752aaf356b26be512629b74687c637a9e34e9fca0ff1b84378"},"operator_account":{"data":"GetResponse","last_update":1651079195,"value":"253cd48df00db05ada9367657108cbe03dfa100910a95d21ce1209f6c88086fe"},"execution":{"data":"GetResponse","last_update":1651079195,"value":"0x6d75d3c44b77138402bc8c20fb38b4c56f01c0be21dc1e814475aef494408728"},"consensus":{"data":"GetResponse","last_update":1651079195,"value":"0x3d7d1c334da45027f4bcb593bc0381367ef064132b87408dc62dd1d1412dee4d"},"validator_network":{"data":"GetResponse","last_update":1651079195,"value":"0xe5a59ae867416cb580b694dbefa506efe2a5a6be096763e03631095e13a80fba"}}
```

Comment:
'cc @gregnazario, do you see the tooling being able to generate these files? Or were you thinking of something different?